### ISSUE TITLE
Incorrectly setting a path, and then running paths will trigger a panic

### ISSUE THREAD

**Issue by rlewkowicz** [starkandwayne/safe#155]
Tried with a handful of versions of vault and safe. Should be reproducible across them, but my current specific versions are: 
```
OS:
macOS High Sierra: 10.13.6

vault --version
Vault v0.9.6 ('7e1fbde40afee241f81ef08700e7987d86fc7242')

safe --version
safe v0.9.3

```
To reproduce run:

`safe local --memory`

Then:
```
safe set secret a/b/c x=yz
safe paths
```

You'll get a panic:
```
panic: runtime error: index out of range

goroutine 1 [running]:
main.main.func22(0x12f0420, 0x5, 0x1561b68, 0x1, 0x0, 0xc420475c01, 0x107cbe5)
	/tmp/build/01f561ac/gopath/src/github.com/starkandwayne/safe/main.go:1513 +0x5bb
main.(*Runner).Execute(0xc420048d40, 0x12f0420, 0x5, 0x1561b68, 0x0, 0x0, 0x0, 0x0)
	/tmp/build/01f561ac/gopath/src/github.com/starkandwayne/safe/runner.go:116 +0xa2
main.main()
	/tmp/build/01f561ac/gopath/src/github.com/starkandwayne/safe/main.go:2913 +0x30d0
```

`safe tree` will show:
```
.
└── secret
    ├──
    └── handshake
```

I think it's dying on the empty space. I also don't know how to recover from that, as in how to delete the now bad path.

**Comment by jhunt** [comment]
This should be included in safe 0.9.5: https://github.com/starkandwayne/safe/releases/tag/v0.9.5

**Comment by rlewkowicz** [comment]
Cool Beans!
